thanks-bot/bot.js

```diff
@@ -23,13 +23,41 @@ bot.on('ready', function (evt) {
 });
 
 bot.on('message', function (user, userID, channelID, message, evt) {
-    switch(message) {
-        case 'Ассаляму аляйкум':
-            bot.sendMessage({
-                to: channelID,
-                message: 'Уа аляйкум салям',
-            });
-
-            break;
+    const regexp = /\!thanks\b/gmi;
+    const bot_user_id = '523928478341660702';
+    const allowed_channel = '525249065039036426';
+    const authorId = evt.d.author.id;
+    const authorName = evt.d.author.username;
+
+    // Conditions for exiting a function
```

Comment:
The function is known, so you have to use a definitive article.
```suggestion
    // Conditions for exiting the function
```